sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingClientBuilder.java

```diff
@@ -176,21 +181,30 @@ private HttpPipeline getDefaultHttpPipeline() {
         // Global Env configuration store
         final Configuration buildConfiguration = (configuration == null)
             ? Configuration.getGlobalConfiguration().clone() : configuration;
+        ClientOptions buildClientOptions = (clientOptions == null) ? new ClientOptions() : clientOptions;
```

Comment:
All the comments for FRClientBuilder apply to the training client builder as well.